misumisumi/nixos-k8s-config
# refactor

- add: add `apps` and `system` dir
- style: nixosの設定のディレクトリ構造を整理
- delete: move `lxd` to `instances/default.nix`
- refoctor: ディレクトリ構造の変更
- add: nixd config
- add: disko support (early project)
- fix: overlay
- add: nixos-anywhere
- fix: disko config
- rename: machines to hosts
- add: ctrl-test
- rename: ctrl to primary
- add: some pkgs
- fix: nixos-anywhereをnixpkgsからではなくリポジトリからに変更
- fix: full support nixos-anywhere
- add: secondary and teriary machine
- add: default initialHashedPassword
- style: zfs等の共通設定をまとめた
- change: lxd-nixos is already old. Moved to nixos-generators.
- fix: rootfs
- change: default vi mode for bash
- change: LXDのpoolとnetworkの設定を永続化できるように分離
- feat: sepalate config of nixos and colmena fix: k8s node configs
- fix: development settings
- update: rootfs config of third node
- fix: making image for lxc
- fix: fix meta for colmena
- feat: managing physical node by using colmena
- update: remove "cardinal" tag
- fix: builtins.toPath is DEPRECATED so replace to "/. +"
- update: rename scripts
- update: mv mkimg4lxc to setup-lxd
- update: Enabled initrd.network.ssh and improved experience of managing hosts in colmena
- fix: fix parse_params
- fix: fix bug of colmena setting
- fix: remove lib.trace
- fix: fix absolute path
- fix: macaddress
- fix: setup-lxd list
- fix: lxd init setting
- fix: create 2 file for terraform output and terraform show
- update: terraform setting for nfs
- add: check-disk-size script
- fix: "nohibernate" is kernelParams
- add: btrfs-progs
- delete: worker
- primary: fix rootfs and additionalfs
- secondary, tertiary: fix rootfs
- fix: fix utils and add diskoConfigrations
- update: flake.lock
- remove: managed by group
- add: additionalfs config
- update: flake
- README: update
- add: github workflows
- statix: format
